src/solc_0.5/Land/erc721/ERC721BaseTokenV2.sol

```diff
@@ -399,13 +461,27 @@ contract ERC721BaseTokenV2 is ERC721Events, SuperOperatorsV2, MetaTransactionRec
         _burn(from, owner, id);
     }
 
+    /**
+     * @param operator Sender of the tx
+     * @param from Owner of the token
+     * @param to Recipient
+     * @param tokenId Token id
+     * @param _data extra data
+     */
     function _checkOnERC721Received(address operator, address from, address to, uint256 tokenId, bytes memory _data)
         internal returns (bool)
     {
         bytes4 retval = ERC721TokenReceiver(to).onERC721Received(operator, from, tokenId, _data);
         return (retval == _ERC721_RECEIVED);
     }
 
+    /**
+     * @param operator Sender of the tx
+     * @param from Owner of the token
+     * @param to Recipient
+     * @param ids Token ids
+     * @param _data extra data
```

Comment:
@returns and @notice are missing for a lot of functions, are we skipping that?

Reply:
@notice are not on internal functions
i'll add @returns tho and something @dev if needed